crates/holochain_conductor_api/src/app_interface.rs

```diff
@@ -84,6 +84,13 @@ pub enum AppRequest {
     ///
     /// [`AppResponse::Ok`]
     ProvideMemproofs(MemproofMap),
+
+    /// Enable the app. Can only be called while the app is in the `MemproofsProvided` state.
+    ///
+    /// # Returns
+    ///
+    /// [`AppResponse::Ok`]
+    EnableAfterMemproofsProvided,
```

Comment:
I wonder if this should just be "Enable" and leave it to documentation for when this can be called? That leaves this open to behavior modification without having to add another EnableXXX later or make a breaking API change to rename it

Reply:
I'm OK with that -- should it be called `EnableApp`?

Reply:
Yes it should probably match, `EnableApp` sounds good